components/dashboard/src/teams/TeamUsage.tsx

```diff
@@ -188,6 +189,19 @@ function TeamUsage() {
                                         <span>{calculateTotalUsage()} Credits</span>
                                     </div>
                                 </div>
+                                <div className="flex flex-col truncate">
+                                    <div className="text-sm text-gray-500">
+                                        Usage is currently in{" "}
+                                        <PillLabel type="warn" className="font-semibold mt-2 py-0.5 px-1 self-center">
+                                            Beta
+                                        </PillLabel>{" "}
+                                    </div>
+                                    <div>
+                                        <a href="#" class="gp-link">
```

Comment:
**issue(blocking)**: Before merging this, make sure there's a public feedback issue that we link to. 💡